### DISCUSSION TITLE
Esta semana en Rust #8

### DISCUSSION THREAD

**Discussion by giscus** [RustLangES/blog#28] [Comentarios de Artículos]
# Esta semana en Rust #8

El crate de esta semana es tokio-graceful, una biblioteca para el apagado correcto de servidores asíncronos basados en tokio. - By @

https://rustlanges.github.io/blog/articles/2023-11-29-this-week-in-rust.html

**Comment by Phosphorus-M** [comment]
Voy a hacer una prueba si esto funciona.
